dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -698,7 +698,7 @@ Spdp::handle_participant_data(DCPS::MessageId id,
       if (from_relay) {
         tport_->write_i(guid, iter->second.local_address_, SpdpTransport::SEND_RELAY);
       } else {
-        tport_->shorten_local_sender_delay_i();
+        tport_->zero_local_sender_delay_i ();
```

Comment:
This could simply be `tport_->write_i(guid, iter->second.local_address_, SpdpTransport::SEND_LOCAL);`

Reply:
Changed